python/data_extraction/export_data.py

```diff
@@ -9,17 +9,33 @@
 with open('config/data_export_fields.json') as json_data_file:
     configuration = json.load(json_data_file)
 
+
 content_links_generator = content_export.content_links_generator(
     blacklist_document_types=configuration['blacklist_document_types']
 )
 
+
+slicer = functools.partial(content_export.content_hash_slicer,
+                           base_fields=configuration['base_fields'],
+                           taxon_fields=configuration['taxon_fields'],
+                           ppo_fields=configuration['ppo_fields'])
+
+
 get_content = functools.partial(content_export.get_content,
-                                base_fields=configuration['base_fields'],
-                                taxon_fields=configuration['taxon_fields'],
-                                ppo_fields=configuration['ppo_fields'],
                                 content_store_url=plek.find('draft-content-store'))
 
 
+untagged_content_links_generator = content_export.content_links_generator(
+    additional_search_fields={'filter_part_of_taxonomy_tree': '_MISSING'},
```

Comment:
As discussed, this isn't ideal as Rummager doesn't have the associations of non Whitehall content with draft taxons.

Reply:
fixed that in a pretty comprehensive rewrite